workers/unity/Assets/Gdk/Core/Tests/Editmode/Components/OptionTests.cs

```diff
@@ -0,0 +1,179 @@
+using System;
+using NUnit.Framework;
+
+namespace Improbable.Gdk.Core.EditmodeTests
+{
+    [TestFixture]
+    public class OptionTests
```

Comment:
Add some tests with reference types too!

Reply:
Added tests for reference types.